Shell/main.cpp

```diff
@@ -655,7 +655,7 @@ static bool handle_builtin(int argc, const char** argv, int& retval)
 
 class FileDescriptionCollector {
 public:
-    FileDescriptionCollector() {}
+    FileDescriptionCollector() { }
```

Comment:
I feel like I see this every time
has the project .clang-format changed without me noticing?

Reply:
On my end, formatting removes the extra space. Maybe it's a clang-format version issue ?

I'm using the `clang-format-9 --style=file -i Shell/main.cpp` command directly.

My clang version is: `9.0.0-2~ubuntu18.04.2 (tags/RELEASE_900/final)`

Reply:
it's clang-format 10.0.0 here.

lemme see if I can grab a build of clang-format 9 to test with

Reply:
it's `SpaceInEmptyBlocks` and the option is added in clang-format 11
:facepalm: